draft-ietf-tls-subcerts.md

```diff
@@ -339,28 +303,78 @@ to the TLS stack, which has the advantage of avoiding changes to
 security-critical and often delicate PKI code (though of course moves that
 complexity to the TLS stack).
 
+## Client and Server behavior
+
+This document defines the following extension code point.
+
+~~~~~~~~~~
+   enum {
+     ...
+     delegated_credential(TBD),
+     (65535)
+   } ExtensionType;
+~~~~~~~~~~
+
+A client which supports this specification SHALL send an empty
+"delegated_credential" extension in its ClientHello.  If the client receives a
+delegated credential without indicating support, then the client MUST abort with
+an "unexpected_message" alert.
+
+If the extension is present, the server MAY send a delegated credential
+extension; if the extension is not present, the server MUST NOT send a delegated
+credential.  A delegated credential MUST NOT be provided unless a Certificate
+message is also sent.  The server MUST ignore the extension unless TLS 1.3 or
+later is negotiated.
```

Comment:
or a later version of TLS is negotiated.

Reply:
Done.